test/e2e/storage/drivers/in_tree.go

```diff
@@ -1441,16 +1443,16 @@ func InitAzureDriver() testsuites.TestDriver {
 	}
 }
 
-func (a *azureDriver) GetDriverInfo() *testsuites.DriverInfo {
+func (a *azureDiskDriver) GetDriverInfo() *testsuites.DriverInfo {
 	return &a.driverInfo
 }
 
-func (a *azureDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
+func (a *azureDiskDriver) SkipUnsupportedTest(pattern testpatterns.TestPattern) {
 	framework.SkipUnlessProviderIs("azure")
```

Comment:
Do we need to skip the AzureFile test as well here ? Or would the future PR on azure file cover this ?

Reply:
azurefile e2e test would be covered by https://github.com/kubernetes-sigs/azurefile-csi-driver/pull/129, @kkmsft, actually the e2e tests in `in_tree.go` do not have lots of coverage.